packages/loopring_v3/contracts/core/iface/ExchangeData.sol

```diff
@@ -48,6 +48,7 @@ library ExchangeData
     struct AuxiliaryData
     {
         uint  txIndex;
+        bool  approved;
```

Comment:
Maybe we should merge `txIndex` and `preApproved` into one single `unit` field.

Reply:
I don't think that will do much. It may save a very small bit of gas in calldata (zeros, which may also be compressed away), but then we have to pay a small gas cost to encode/decode the 2 values. Both are tiny gas costs. So I think gas savings aren't really there, so cleaner I think not to do so.